jni_gen/examples/in_app_java/pubspec.yaml

```diff
@@ -0,0 +1,67 @@
+name: in_app_java
+description: |
+  Example on how to use jnigen to bridge between java and dart in a flutter
+  project.
+
+publish_to: 'none' # Remove this line if you wish to publish to pub.dev
+
+version: 1.0.0+1
+
+environment:
+  sdk: ">=2.17.6 <3.0.0"
+
+dependencies:
+  flutter:
+    sdk: flutter
+  jni:
+    path: ../../../jni/
+
+  # The following adds the Cupertino Icons font to your application.
+  # Use with the CupertinoIcons class for iOS style icons.
+  cupertino_icons: ^1.0.2
+
+dev_dependencies:
+  flutter_test:
```

Comment:
```
$ dart pub get
Resolving dependencies... 
Because pdfbox_plugin_example depends on flutter_test from sdk which doesn't exist (the Flutter SDK is not available), version solving failed.

Flutter users should run `flutter pub get` instead of `dart pub get`.
```